README.md

```diff
@@ -0,0 +1,54 @@
+# Zabbix Exporter
+
+This project is a fork of [original zabbix exporter](https://github.com/MyBook/zabbix-exporter) 
+with advances functionality: 
+filtering was added for each zabbix metric with configuration-defined hosts and item names, which significantly 
+improves execution time.
+
+### Usage example
+```shell
+Usage: zabbix_exporter [OPTIONS]
+  Zabbix metrics exporter for Prometheus
+
+  Use config file to map zabbix metrics names/labels into prometheus. Config
+  below transforms this:
+      local.metric[uwsgi,workers,myapp,busy] = 8
+      local.metric[uwsgi,workers,myapp,idle] = 6
+
+  into familiar Prometheus gauges:
+      uwsgi_workers{instance="host1",app="myapp",status="busy"} 8
+      uwsgi_workers{instance="host1",app="myapp",status="idle"} 6
+
+  YAML config example:
+      metrics:
+        - key: 'local.metric[uwsgi,workers,*,*]'
+          name: 'uwsgi_workers'
+          labels:
+            app: $1
+            status: $2
+          reject:
+            - 'total'
+          hosts:
+            - name.of.host.1
+            - name.of.host.2
+          item_names:
+            - '*item.name.substr.1*'
+            - '*item.name.substr.2*'
```

Comment:
`type:` parameter should be added to example

Reply:
done